core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -306,6 +355,13 @@ public Options readConsistency(ReadConsistency readConsistency) {
      */
     Options forTablet(KeyExtent extent);
 
+    /**
+     * Get the tablet metadata for the given extents. This will find tablets based on end row, so
+     * its possible the prev rows could differ for the tablets returned. If this matters then it
```

Comment:
"its" -> "it's" ("it is")

```suggestion
     * it's possible the prev rows could differ for the tablets returned. If this matters, then it
```